src/qibolab/_core/instruments/emulator/emulator.py

```diff
@@ -61,6 +61,8 @@ def play(
         options: ExecutionParameters,
         sweepers: list[ParallelSweepers],
     ) -> dict[int, Result]:
+        # convert align to delays
+        sequences = [seq.align_to_delays() for seq in sequences]
```

Comment:
To avoid shadowing, just append an underscore
```suggestion
        sequences_ = [seq.align_to_delays() for seq in sequences]
```

(if you can come with a more expressive name, which in this case could be something like `deterministic_sequences`, or `align_unraveled_sequences`, ... it is just better - but I also do not like these names, because are not that clear, and verbose - since it is just used right after, appending an underscore it is a poor but acceptable solution)